.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation with Sphinx
+sphinx:
+   builder: html
+   configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats: []
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+   version: "3.8"
```

Comment:
this is the python version readthedocs build process will use when running sphinx-build ... so anything recent is ok. 

It would make sense to keep in sync with the one `tox -e docs` uses -- but currently we don't set a default python for tox I think. In that light maybe not setting the version here either would be best?

Reply:
Yes, the Python version used by Tox isn't set explicitly. We use the version on the host and run our CI in GitHub runners with specific versions installed. Not setting here makes sense in that context, I think.

Reply:
Seems like by default read the docs uses the biggest version from this list:
https://docs.readthedocs.io/en/stable/config-file/v2.html#build-image meaning 3.7 and 3.8 for stable and latest

Reply:
It is removed: 69c2270